src/github.ts

```diff
@@ -190,4 +195,84 @@ export class GitHub {
       url: artifactURL
     };
   }
+
+  public static async writeBuildSummary(opts: BuildSummaryOpts): Promise<void> {
+    // can't use original core.summary.addLink due to the need to make
+    // EOL optional
+    const addLink = function (text: string, url: string, addEOL = false): string {
+      return `<a href="${url}">${text}</a>` + (addEOL ? os.EOL : '');
+    };
+
+    const refsSize = Object.keys(opts.exportRes.refs).length;
+
+    // prettier-ignore
+    const sum = core.summary
+      .addHeading('Docker Build summary', 1)
+      .addRaw(`<p>`)
+        .addRaw(`For a detailed look at the build, download the following <code>.dockerbuild</code> file and import it into Docker Desktop's Builds view. `)
+        .addBreak()
+        .addRaw(`The build file provides insights into your build, such as timing, dependencies, results, logs, traces, and more. `)
+        .addRaw(addLink('Learn more', 'https://docs.docker.com/desktop/use-desktop/builds/#import-build-records'))
+      .addRaw('</p>')
+      .addRaw(`<p>`)
+        .addRaw(`:arrow_down: ${addLink(`<strong>${opts.uploadRes.filename}</strong>`, opts.uploadRes.url)} (${Util.formatFileSize(opts.uploadRes.size)})`)
+        .addBreak()
+        .addRaw(`This file includes <strong>${refsSize} build record${refsSize > 1 ? 's' : ''}</strong>.`)
+      .addRaw(`</p>`)
+      .addRaw(`<p>`)
+        .addRaw(`Was this useful? `)
```

Comment:
I'm not sure about the past tense here - this text appears before the summary. Maybe "Find this useful?" could work?
```suggestion
        .addRaw(`Find this useful? `)
```

Reply:
Yes agree!